core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -105,14 +83,14 @@
    */
   public static final String FIELD_COUNT_PROPERTY_DEFAULT = "10";
 
-  int fieldcount;
+  private int fieldcount;
```

Comment:
All the ``private``s should be ``protected``s for folks overloading the class (like me :)